src/rpdk/core/data/schema/cloudformation-resource-type-configuration-schema.v1.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
We should not release this with Public Registry launch. For more info check with @rajdivya

Reply:
Ack